### ISSUE TITLE
uninstall process 

### ISSUE THREAD

**Issue by andreas83** [thedevdojo/voyager#874]
i would love to use your voyager package on a ongoing project.
but when i tried to install it, i got the error, that the post table already exists.

second trouble was the uninstall process, i saw quite a lot migrations and seed files, would love to see a uninstall task / process...

**Comment by thinkdj** [comment]
+1 
Need an uninstall.

**Comment by sepehr09** [comment]
how we can uninstall voyager ?

**Comment by ShahidKhosa** [comment]
Hare are the steps to remove voyager. I also installed the voyager front-end.

1. composer remove pvtl/voyager-frontend

2. composer remove tcg/voyager

3. Remove these routes from web.php file 
   Route::group(['prefix' => 'admin'], function () {
     Voyager::routes();
 });

4. Remove the parent class from app/User.php file and extend it from "Authenticatable". Below is the example how the user.php file look like.

use Illuminate\Foundation\Auth\User as Authenticatable;
class User extends Authenticatable

**Comment by mtslzr** [comment]
I believe you'll also wanna remove the hook repo from `composer.json` that gets added with Voyager:

```json
        "hooks": {
            "type": "composer",
            "url": "https://larapack.io"
        }
```

**Comment by tnylea** [comment]
This will be implemented in Version 2.0 👍 Adding it to the 2.0 planning project: https://github.com/the-control-group/voyager/projects/6
